src/index.ts

```diff
@@ -136,9 +138,12 @@ function requestToFetchOptions(reqOpts: Options) {
     process.env.HTTPS_PROXY ||
     process.env.https_proxy;
   if (reqOpts.proxy || proxy) {
+    // TODO: we should support an HTTP proxy.
```

Comment:
So there is an `http` version of the proxy agent by the same maintainer as the https one ([`http-proxy-agent`](https://www.npmjs.com/package/http-proxy-agent)).